code/datums/tips/tips.dm

```diff
@@ -98,7 +101,38 @@ GLOBAL_LIST_EMPTY(tips_by_category)
 		"There is no escape.",
 		"Most large kitchen appliances 3D print their own single-use dishes out of leftover food scraps! Yum yum, recycling!",
 		"As a Clown, you are probably on the wrong server.",
-		"If you say the Q-word, anything bad that happens this shift is your fault."
+		"If you say the Q-word, anything bad that happens this shift is your fault.",
+		"Never gonna give you up, never gonna let you down. Never gonna run around and desert you. Never gonna make you cry, never gonna say goodbye, never gonna tell a lie, and hurt you.",
+		"Ginny is the best department pet.",
+		"Ian is the best department pet.",
+		"Crusher is the best department pet.",
+		"Columbo is the best department pet.",
+		"Don't lie to yourself, when was the last time a Research director took Ives out?",
+		"Chauncey is the best pet.",
+		"Remember, live wires want to hold hands with you.",
+		"A tip or something... Whatever...",
+		"500 Cigarettes.",
+		"Remember, you can touch grass in real life.",
+		"If you walk without rhythm, you won't attract the phoron worm.",
+		"Ough!",
+		"Rubber ducks are just reskinned bike horns. There's two of them on the ship by default.",
+		"As a chef, you can oven when you of in the cold food of out hot eat the food!",
+		"As a Sturgeon, you are a fresh water fish.",
+		"As an Engineer, you can pick up your Sentry Gun, Dispenser, and Teleporters with right click.",
+		"As a Vaurca Consular, you're a big bug.",
+		"Would you like to sign my petition?",
+		"I bought WinRAR, did you?",
+		"Extended voters always win, the game was rigged from the start.",
+		"#BringBackVirology",
+		"You are not Literally Him, you're just dehydrated.",
+		"You should watch Tasting History with Max Miller",
```

Comment:
remove this, should not reference... irl youtubers? really?

Reply:
References in OOC are permitted, tips of the round is before the round even starts

Reply:
I don't think this one should be removed. OOC references are fine, as far as I can tell. Its before the round starts anyways.

Reply:
Remove it. I see no valid reason to add some random youtuber reference to the code, neither IC descriptions nor OOC tips. Nor any youtubers, celebrities, nor any other real life personas.

Putting it there means the server (staff/devs/playerbase) as a whole endorses that particular random youtuber, and I see no reason why we would do that.

Otherwise, if this is fine, should we allow anyone to add references to any random youtubers or whatever that they like? Or if someone does not like this specific youtuber, should they be able to open up a PR to remove it?

Reply:
Very fair point and good explanation, removing it now. Thank you :)